content/asset-tracking/using-the-sdks.textile

```diff
@@ -600,8 +601,82 @@ subscriber.nextLocationUpdateIntervals
 
 See also the SDK reference docs for more "Subscriber properties":https://sdk.ably.com/builds/ably/ably-asset-tracking-android/main/dokka/subscribing-sdk/subscribing-sdk/com.ably.tracking.subscriber/-subscriber/index.html.
 
+h2(#location-animator). Location animator
+
+The location animator is an extension for the subscriber that provides smooth location updates animation. 
+
+The location animator receives location updates from the publisher and creates animation steps from them. These steps are then applied to an ongoing animation. 
```

Comment:
"The location animator receives location updates from the publisher" - doesn't it imply that the location animator somehow communicates directly with the Publisher? 🤔 I wonder whether we should change it to show that you as the user have to pass the locations from the subscriber to the animator?
```suggestion
The location animator creates animation steps from location updates that are provided by the Subscriber SDK. These steps are then applied to an ongoing animation. 
```